pkg/kubelet/types/constants.go

```diff
@@ -16,8 +16,22 @@ limitations under the License.
 
 package types
 
+import "runtime"
+
 const (
-	PodInfraContainerImage = "gcr.io/google_containers/pause:2.0"
+
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
+
+	DefaultPodInfraContainerImageName = "gcr.io/google_containers/pause"
```

Comment:
So where should I put it? `cmd/kubelet/app/constants.go`, `cmd/kubelet/app/pause.go` or `cmd/kubelet/app/options/options.go`, which do you prefer?

Another option would be breaking out all pause constants to some other place not directly tied to `kubelet`.